.github/workflows/pylint.yml

```diff
@@ -0,0 +1,23 @@
+name: Pylint
```

Comment:
This file is boilerplate supplied by GitHub for adding a check to block PRs if they don't pass lint.

Reply:
Looks cool, maybe on our next call we can talk about how this works in tandem with the requirements.txt file.

Reply:
Sure thing. I love that.